okhttp/src/main/java/okhttp3/internal/platform/ConscryptPlatform.kt

```diff
@@ -41,7 +41,9 @@ class ConscryptPlatform private constructor() : Platform() {
       SSLContext.getInstance("TLS", provider)
 
   override fun platformTrustManager(): X509TrustManager {
-    return Conscrypt.getDefaultX509TrustManager()
+    return Conscrypt.getDefaultX509TrustManager().also {
```

Comment:
We do this proactively because TrustManagerBridge will wrap this call, so it's hard to do otherwise.